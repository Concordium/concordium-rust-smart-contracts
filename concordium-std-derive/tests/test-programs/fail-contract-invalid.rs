//! Ensure that the macros generate compilable code.

use concordium_contracts_common::*;
use concordium_std_derive::*;

const CONTRACT: ContractAddress = contract_address!("<0, 0>");

fn f() {
    println!("{:?}", CONTRACT.to_string());
}

fn main() {}
