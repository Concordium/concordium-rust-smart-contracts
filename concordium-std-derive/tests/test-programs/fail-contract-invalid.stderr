error: Invalid contract address: Failed to parse the subindex integer: invalid digit found in string
 --> tests/test-programs/fail-contract-invalid.rs:6:53
  |
6 | const CONTRACT: ContractAddress = contract_address!("<0, 0>");
  |                                                     ^^^^^^^^
